ChangeLog

```diff
@@ -1,3 +1,64 @@
+Changes to squid-5.0.1 (08 Jan 2020):
```

Comment:
What is "squid-5.0.1"? I see no corresponding git tag. Judging by the number of entries in this list, the v5.0.1 tag would have to be pointing to some commit made long time ago, which does not make sense. Perhaps this should say 

```suggestion
Changes in squid-5.0.1 (08 Jan 2020):
```

or

```suggestion
Changes up to squid-5.0.1 (08 Jan 2020):
```


instead?

Reply:
I edited the previous comment to make the precondition for my suggestions explicit and to express a preference for the first suggestion.

Reply:
The text is what Duane picked for use. It only matter to software parsing our ChangeLog.

In regards to the tags, the backport commit of this PR to the v* branch in the squidadm repository is where the tag is applied and the release bundle is generated from that commit.